mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/notation/CommonNotationCombinators.kt

```diff
@@ -18,7 +18,14 @@ object CommonNotationCombinators {
   val row = StringCombinators.digit().map { 8 - it }.filter { it in 0 until Board.Size }
 
   /** A [Parser] which returns a [Position]. */
-  val position = column.flatMap { x -> row.map { y -> Position(x, y) } }.filter { it.inBounds }
+  val position = this.computePosition(column, row)
+
+  fun computePosition(
+      column: Parser<String, Int>,
+      row: Parser<String, Int>
+  ): Parser<String, Position> {
+    return column.flatMap { x -> row.map { y -> Position(x, y) } }.filter { it.inBounds }
+  }
```

Comment:
Missing documentation

Reply:
Fix in https://github.com/epfl-SDP/android/pull/311/commits/e5fcc2f8139de2c76ae277772d66f7851fa362b8